src/app.js

```diff
@@ -0,0 +1,69 @@
+import DotNet from "./dot_net";
+
+let generators = {
+  ".NET": DotNet
+};
+
+// console.log(generators);
+
+const DATA_MODEL_OUTPUT_DIR = "../OpenActive.NET/";
+
+const program = require("commander");
+
+program.command("list").action(() => {
+  let keys = Object.keys(generators);
+  console.log("Available languages: ", keys);
+});
+
+program
+  .command("generate <language>")
+  .option("--no-beta", "Disable the beta extension")
+  .option("-d, --destination <destination>", "Output directory")
+  .action((language, options) => {
+    console.log(options);
+
+    if (!options.destination) {
+      console.error("Destination must be specified");
+      return;
+    }
+
+    if (!language) {
+      console.error("Language must be specified");
+      return;
+    }
+
+    const Generator = generators[language];
+    if (!language) {
```

Comment:
If language not supported check Generator instead of language (to avoid crash):
```suggestion
    if (!Generator) {
```